### DISCUSSION TITLE
Updating without Ctrl + S

### DISCUSSION THREAD

**Discussion by mihalt** [reloadware/reloadium#108] [Q&A]
Hi! Nice software for Pycharm! I would like to ask, do I have any possibilities to reload vars of running without Ctrl + S combination? I would like to see changed results automatically.

**Comment by dkrystki** [comment]
Hi @mihalt , it's not possible now.
I am not convinced about UX of that feature.
I think it would produce a lot unwanted reloads in the middle of development producing frame reload errors.

**Comment by mihalt** [comment]
I would like to have option like this at least. Would be nice to see this
feature in the future

**Reply by ChillarAnand** [reply]
@mihalt There are other plugins or configurations to auto save a file at regular intervals. You can try those in combination with reloadium. That should work too.
